### ISSUE TITLE
Crash when displaying list of repositories with Gravatar disabled

### ISSUE THREAD

**Issue by Matrixcoffee** [maks/MGit#354]
Steps to reproduce:
* Get to a clean slate
  - Force-stop MGit
  - Clear app data
* Enter settings and disable Gravatar
* Clone `https://github.com/Matrixcoffee/matrix-knowledge-base`
* Watch MGit crash as soon as the clone finishes and the repository list is displayed
* Continue to watch MGit crash upon opening (app becomes unusable)

An alternative sequence is to clone first, which appears to work. Disable gravatar and it breaks.

* Android 4.4.2
* MGit versioncode 208 from F-Droid

**Comment by kaeptmblaubaer1000** [comment]
@Matrixcoffee can you please publish the traceback here?

**Comment by Matrixcoffee** [comment]
Finally managed to exfiltrate the stack trace by installing k9mail, which at least lets you skip past connection failures. The default mail client will not work without a properly set up e-mail account.

```
USER_COMMENT=
ANDROID_VERSION=4.4.2
APP_VERSION_NAME=1.5.4.3
BRAND=x
PHONE_MODEL=x
CUSTOM_DATA=
STACK_TRACE=android.content.res.Resources$NotFoundException: File res/drawable/ic_default_author.xml from drawable resource ID #0x7f020072. If the resource you are trying to use is a vector resource, you may be referencing it in an unsupported way. See AppCompatDelegate.setCompatVectorFromResourcesEnabled() for more info.
	at android.content.res.Resources.loadDrawable(Resources.java:2152)
	at android.content.res.Resources.getDrawable(Resources.java:710)
	at com.nostra13.universalimageloader.core.DisplayImageOptions.getImageOnFail(DisplayImageOptions.java:142)
	at com.nostra13.universalimageloader.core.LoadAndDisplayImageTask$2.run(LoadAndDisplayImageTask.java:358)
	at android.os.Handler.handleCallback(Handler.java:808)
	at android.os.Handler.dispatchMessage(Handler.java:103)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:5299)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:825)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:641)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector
	at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:933)
	at android.graphics.drawable.Drawable.createFromXml(Drawable.java:877)
	at android.content.res.Resources.loadDrawable(Resources.java:2148)
	... 12 more
org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector
	at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:933)
	at android.graphics.drawable.Drawable.createFromXml(Drawable.java:877)
	at android.content.res.Resources.loadDrawable(Resources.java:2148)
	at android.content.res.Resources.getDrawable(Resources.java:710)
	at com.nostra13.universalimageloader.core.DisplayImageOptions.getImageOnFail(DisplayImageOptions.java:142)
	at com.nostra13.universalimageloader.core.LoadAndDisplayImageTask$2.run(LoadAndDisplayImageTask.java:358)
	at android.os.Handler.handleCallback(Handler.java:808)
	at android.os.Handler.dispatchMessage(Handler.java:103)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:5299)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:825)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:641)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by kaeptmblaubaer1000** [comment]
Duplicate of #300

**Comment by kaeptmblaubaer1000** [comment]
@Matrixcoffee I've already fixed this on `master`, but @maks hasn't released it yet.

**Comment by maks** [comment]
The release with a fix (1.5.4.4) is now available in the Play Beta channel: https://play.google.com/apps/testing/com.manichord.mgit
